src/procedures/iotscape/iotscape.js

```diff
@@ -268,6 +279,12 @@ async function _mergeWithExistingService(name, service) {
         .map((method) => method.name),
     );
 
+    // Check if methods are the same
```

Comment:
Is it possible for the method names to be the same but the arguments to change? I don't think so but wanted to confirm

Reply:
It's possible, but currently not really worth considering at the moment because it's a pretty messy thing in NetsBlox if there's existing code from the original method. Eventually there might need to be a better system for this, but the current use cases shouldn't have an issue from it (for RO, all the services are predefined).